wmcontrol.py

```diff
@@ -483,14 +489,18 @@ def loop_and_submit(cfg):
                       print "no white listing"
                   if test_mode:
                       print "Finished in", int((time.time()-t)*1000), "\bms"
-
           if params['RunWhitelist']:
               params['Task1']['RunWhitelist'] = params['RunWhitelist']
 
           if 'RunWhitelist' in params: #if params has key we remove it
               params.pop('RunWhitelist') #because it was set as Task parameter
           if 'InputDataset' in params:
               params.pop('InputDataset')
+
+          if params.has_key("LumiList") and params['LumiList']:
+              params['Task1']['LumiList'] = params['LumiList']
```

Comment:
I do believe this will crash if we submit non taskchain, but pass LumiList parameter.